src/components/Ui/Dropdown.vue

```diff
@@ -56,7 +56,7 @@ onBeforeUnmount(() => window.removeEventListener('click', close));
               v-if="item.icon"
               :name="item.icon"
               size="21"
-              class="align-middle mr-2"
+              class="align-middle mr-2 !leading-[0]"
```

Comment:
Would be cleaner if we refactor all the dropdowns to use the slot `<template v-slot:item="{ item }">` like we do on the Home view. Then this can also be removed from inside the Dropdown component 
![image](https://user-images.githubusercontent.com/51686767/152301958-d1fa226e-346b-4cc8-a992-1cf46c46d296.png)

Reply:
Actually maybe not all, only the ones that have more than just text.

Reply:
Done ✅ 

Verified all dropdowns working fine

![image](https://user-images.githubusercontent.com/69431456/152537125-5d2689ec-ff30-4909-a5dd-5ad4cb10a391.png)
![Screenshot 2022-02-04 at 6 59 41 PM](https://user-images.githubusercontent.com/69431456/152537145-9974d78d-e6b5-4765-95bc-901c327a9c24.png)
![Screenshot 2022-02-04 at 6 59 56 PM](https://user-images.githubusercontent.com/69431456/152537171-ae9a9f11-59b4-4f38-914c-59f4881b6987.png)